paket.lock

```diff
@@ -1,17 +1,25 @@
 STORAGE: NONE
 NUGET
   remote: https://www.nuget.org/api/v2
-    dotnet-fable (2.0.2) - clitool: true
+    dotnet-fable (1.3.19) - clitool: true
```

Comment:
@forki This version is incorrect you are back to Fable 1

Reply:
urgs